src/core/graph.test.js

```diff
@@ -1045,40 +995,30 @@ describe("graph", () => {
         const g1 = makeGraph("one");
         const g2 = makeGraph("two");
         expect(() => {
-          Graph.mergeConservative(g1, g2);
-        }).toThrow(/distinct edges with address/);
+          Graph.mergeConservative([g1, g2]);
+        }).toThrow(/edge.*distinct contents/);
+      });
+
+      it("is the identity on a singleton list", () => {
+        const merged = Graph.mergeConservative([demoData.advancedMealGraph()]);
+        expect(merged.equals(demoData.advancedMealGraph())).toBe(true);
       });
 
-      function assertNotCalled(...args) {
-        throw new Error(`called with: ${args.join()}`);
-      }
       it("has the empty graph as a left identity", () => {
-        const merged = Graph.merge(
+        const merged = Graph.mergeConservative([
           new Graph(),
           demoData.advancedMealGraph(),
-          assertNotCalled,
-          assertNotCalled
-        );
+        ]);
         expect(merged.equals(demoData.advancedMealGraph())).toBe(true);
       });
-      it("has the empty graph as a right identity", () => {
-        const merged = Graph.merge(
+
+      it("has the empty graph as a left identity", () => {
```

Comment:
"has the empty graph as a left identity" - two tests of this name now?

Reply:
Fixed.